model-optimizer/extensions/middle/L2NormToNorm.py

```diff
@@ -82,6 +82,17 @@ def replace_pattern(self, graph: Graph, match: dict):
             log.debug('The value of the "maximum_y_data" is not defined or is not constant')
             return
 
+        # We need to check axes which performed reduction because IE supports reduction only along HW and HWC axes.
+        axes = match['sum'].in_port(1).data.get_value()
+        axes = int64_array(axes)
+        if axes.shape == ():
+            axes = int64_array([axes])
+        axes.sort()
+        if not (np.array_equal(axes, int64_array([1, 2])) or np.array_equal(axes, int64_array([1, 2, 3]))):
```

Comment:
to me the most obvious check: np.any(axes == 0)

Reply:
This check was refactored and updated to be more clear and common.